Project-II-Calculator/calculator/src/components/ButtonComponents/ActionButton.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Button.css';
+
+const ActionButton = props => {
+  return (
+    <a onClick={props.click} href="##" className="button actionBtn">{props.value}</a>
```

Comment:
this seems odd. i'm curious about why you chose to use anchor tags for these buttons.